SwiftVoiceCallKitQuickstart/ViewController.swift

```diff
@@ -216,17 +216,17 @@ class ViewController: UIViewController, PKPushRegistryDelegate, TVONotificationD
 
     // MARK: TVONotificaitonDelegate
     func callInviteReceived(_ callInvite: TVOCallInvite) {
-        if (callInvite.state == .pending) {
-            handleCallInviteReceived(callInvite)
-        } else if (callInvite.state == .canceled) {
-            handleCallInviteCanceled(callInvite)
-        }
+        handleCallInviteReceived(callInvite)
```

Comment:
we can probably move the `handleCallInviteReceived` code to here. no need to have handle* method with new APIs now.